### PR TITLE
Add new section

### PR DESCRIPTION
**Pull request by prince314**
Add new section

### CODE REVIEW

File: src/style/default.scss [hunk 1]
```diff
@@ -1320,4 +1275,236 @@ p {
...
+      color: $green;
+    }
+  }
+
+  & .title-content {
+    font-size: 16px;
+    font-weight: $light;
+    padding-bottom: 80px;
+    text-align: center;
+  }
```
Comment (lazydeveloperme) [inline src/style/default.scss]: Using & before CSS specificity is not required

File: src/style/_mixin.scss [hunk 2]
```diff
@@ -19,4 +19,71 @@
...
+
+@mixin flex-column-center {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin color-plane-per {
+  color: white;
```
Comment (lazydeveloperme) [inline src/style/_mixin.scss]: Better to use $white variable for consistency

File: src/index.html [hunk 3]
```diff
@@ -433,11 +433,143 @@ <h3>Achmad Doe</h3>
...
+          <p>300 DPI High Resolution</p>
+          <p>Print Ready</p>
+          <p>PDF & AI Format</p>
+          <p>Free 1 year Domain</p>
+          <p>10 GB Free Hosting</p>
+          <p>Unlimmited Revision</p>
+          <p>3 Months Support</p>
+          <button>ORDER NOW</button>
+        </div>
+      </article>
```
Comment (lazydeveloperme) [inline src/index.html]: Color pane and white pane should be in single article

File: src/style/default.scss [hunk 4]
```diff
@@ -1320,4 +1275,236 @@ p {
...
+        height: 325px;
+        background-color: $lighter-blue;
+        margin: 0px 10px 0 10px;
+
+        @include flex-column-center;
+
+        @include color-plane-per;
+      }
+
+      & .down {
```
Comment (lazydeveloperme) [inline src/style/default.scss]: Using ampersand before specificity is not required

File: src/style/default.scss [hunk 5]
```diff
@@ -23,8 +24,8 @@ p {
   #navbar-trigger {
     display: none;
     font-size: 24px;
-    color: $grey;
-    background-color: white;
+    color: white;
```
Comment (happimaker) [inline src/style/default.scss]: White color should be another variable.

File: src/style/default.scss [hunk 6]
```diff
@@ -888,28 +897,6 @@ p {
...
-
-
-
-
-
-
-
-
-
-
```
Comment (happimaker) [inline src/style/default.scss]: What are the empty lines?
You should follow the coding style.

File: src/style/default.scss [hunk 7]
```diff
@@ -1320,4 +1275,236 @@ p {
...
+        @include common-button (white, $lighter-blue, 2px);
+        display: none;
+      }
+
+      @media (max-width: $sm) {
+        flex-direction: column !important;
+      }
+
+      @media (max-width: $lg) {
+        margin: 10px 0 10px 0;
```
Comment (happimaker) [inline src/style/default.scss]: Same as :
margin: 10px 0;
simply.

**Pull request merged**